light-zk.js/tests/transactionParameters.test.ts

```diff
@@ -433,13 +436,17 @@ describe("Transaction Parameters Functional", () => {
         verifierIdl: VERIFIER_IDLS[j],
       });
       assert.equal(params.action.toString(), Action.UNSHIELD.toString());
-      assert.equal(
-        params.publicAmountSpl.sub(FIELD_SIZE).mul(new BN(-1)).toString(),
-        depositAmount.toString(),
+      assert(
+        params.publicAmountSpl
+          .sub(FIELD_SIZE)
+          .mul(new BN(-1))
+          .eq(depositAmount),
```

Comment:
I switched from assert to assert.equal because chai assert does not show you the values it failed with while assert.equal prints both values on failure see below:

```
> chai.assert(1 === 2)
Uncaught [AssertionError: Unspecified AssertionError] {
  showDiff: false,
  actual: null,
  expected: undefined
}
> chai.assert.equal(1, 2)
Uncaught [AssertionError: expected 1 to equal 2] {
  showDiff: true,
  actual: 1,
  expected: 2,
  operator: 'strictEqual'
}
```